apollo-api/api/apollo-api.api

```diff
@@ -328,14 +328,15 @@ public final class com/apollographql/apollo3/api/CompiledVariable {
 public final class com/apollographql/apollo3/api/CustomScalarAdapters : com/apollographql/apollo3/api/ExecutionContext$Element {
 	public static final field Empty Lcom/apollographql/apollo3/api/CustomScalarAdapters;
 	public static final field Key Lcom/apollographql/apollo3/api/CustomScalarAdapters$Key;
-	public synthetic fun <init> (Ljava/util/Map;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Ljava/util/Map;Lcom/apollographql/apollo3/api/Executable$Variables;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
This is the same issue as https://github.com/Kotlin/binary-compatibility-validator/issues/73.

This constructor is private so the breaking change should be ok here